git_theta/lsh/e2lsh.py

```diff
@@ -0,0 +1,89 @@
+"""Classes for computing Euclidean locality-sensitive hashes"""
```

Comment:
I can't do a file level comment on the `.npy` files, but we should include a readme with info about what they are, i.e. why we have them, their shapes, their dtypes, the seed used to make them, etc.

Reply:
The files aren't needed anymore since we can consistently generate seeded random values